exercise_06/checkCollectionHasElements.js

```diff
@@ -3,3 +3,30 @@
  * input: array (collection)
  * output: boolean
  */
+/**
+ * 
+ * @param {} collection - array 
+ * @returns 
+ */
+
+/** function checkCollectionHasElements(collection){
```

Comment:
This fn is commented but exported, so this is causing an error in preview deploy.

Did you check logs i npreview deploy? 🤔

- [ ] Fix error